packages/web-components/fast-components-msft/src/card/card.styles.ts

```diff
@@ -13,6 +14,16 @@ export const CardStyles = css`
         background: ${neutralLayerCardBehavior.var};
         border-radius: calc(var(--elevated-corner-radius) * 1px);
         ${elevation};
-        border: calc(var(--outline-width) * 1px) solid transparent;
     }
-`.withBehaviors(neutralLayerCardBehavior);
+`.withBehaviors(
+    neutralLayerCardBehavior,
+    forcedColorsStylesheetBehavior(
+        css`
+            :host {
+                forced-color-adjust: none;
+                background: ${SystemColors.Canvas};
+                box-shadow: 0 0 0 1px ${SystemColors.CanvasText};
+            }
+        `
+    )
```

Comment:
This looks like a feature - should we call out that you're including high contrast styles here?

Reply:
Should these be seperate PR's? Once appears to be a fix, the other appears to be a feature.

Reply:
I put them in one as to not regress the high contrast work when fixing the padding bug. But I am all for making them separate PRs.